Models/RegistrationBase.php

```diff
@@ -359,12 +363,108 @@ public static function create($request)
         $reg->fill($inputs);
         $reg->save();
 
+
         $response['success'] = true;
         $response['data']['item'] = $reg;
         $response['messages'][] = trans('vaahcms-general.saved_successfully');
         return $response;
 
     }
+
+    //-------------------------------------------------
+
+    public static function createRegistration($request)
+    {
+
+        $inputs = $request->all();
+
+           $rules = [
+                'first_name' => 'required|max:150',
+                'last_name' => 'required|max:150',
```

Comment:
@we-anil-y001 it should not be required

Reply:
Remove both first_name and last_name from required.